### PR TITLE
feat: add SMSL link, update code for SMSL

### PR DESCRIPTION
**Pull request by wildintellect**
Needed to make several changes to the notebook for compatibility with SageMaker Studio Lab #4 

- Ask for AWS creds to do cross region Requester Pays,
- cleanup imports for SMSL that doesn't rely on conda, using pip for now, removing stackstac, folium, geopandas, adding boto3

TODO:
- [x] Remove references to Pangeo Notebook
- [x] Add to Readme warnings about running in same region when scaling analysis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,16 @@ Jupyter Notebook examples related to the datasets in the [Amazon Sustainability
 
 In particular, these notebooks demonstrate how to query and access data from a [Spatial Temporal Asset Catalog (STAC)] of the open datasets that have been cataloged.
 
-The notebooks are designed to work with [AWS SageMaker Studio Lab](https://studiolab.sagemaker.aws/) which you can use for free[^1], but should also work with other Jupyter Notebook environments. You may need to be in the same AWS region as the dataset you want to access for it to work.
+The notebooks are designed to work with [AWS SageMaker Studio Lab](https://studiolab.sagemaker.aws/) which you can use for free[^1], but should also work with other Jupyter Notebook environments. You may need to be in the same AWS region as the dataset, or proide an AWS account for requester pays.
```
Comment (jjfrench) [inline README.md]:
```suggestion
The notebooks are designed to work with [AWS SageMaker Studio Lab](https://studiolab.sagemaker.aws/) which you can use for free[^1], but should also work with other Jupyter Notebook environments. You may need to be in the same AWS region as the dataset, or provide an AWS account for requester pays.
```

File: README.md [hunk 2]
```diff
@@ -3,11 +3,16 @@ Jupyter Notebook examples related to the datasets in the [Amazon Sustainability
...
+
 
 ## Datasets
 
 * Copernicus Digital Elevation Model (DEM) [Glo-30 (30m) AWS Open Data](https://registry.opendata.aws/copernicus-dem/) <a href="https://studiolab.sagemaker.aws/import/github/https://github.com/developmentseed/asdi-examples/blob/main/examples/cop-dem-glo-30-stac-access.ipynb" rel="nofollow"><img src="https://camo.githubusercontent.com/8c5378ff3bf6f71a57442940234293bd63c7ed2418d64f74f2bda3dc6f2904ed/68747470733a2f2f73747564696f6c61622e736167656d616b65722e6177732f73747564696f6c61622e737667" alt="Open In SageMaker Studio Lab" data-canonical-src="https://studiolab.sagemaker.aws/studiolab.svg" style="max-width: 100%;"></a>
+
+* Sentinel 1 GRD []() <a href="https://studiolab.sagemaker.aws/import/github/https://github.com/developmentseed/asdi-examples/blob/main/examples/sentinel-1-stac-example.ipynb" rel="nofollow"><img src="https://camo.githubusercontent.com/8c5378ff3bf6f71a57442940234293bd63c7ed2418d64f74f2bda3dc6f2904ed/68747470733a2f2f73747564696f6c61622e736167656d616b65722e6177732f73747564696f6c61622e737667" alt="Open In SageMaker Studio Lab" data-canonical-src="https://studiolab.sagemaker.aws/studiolab.svg" style="max-width: 100%;"></a>
```
Comment (jjfrench) [inline README.md]:
```suggestion
* Sentinel 1 GRD [Sentinel-1 GRD AWS Open Data](https://registry.opendata.aws/sentinel-1/) <a href="https://studiolab.sagemaker.aws/import/github/https://github.com/developmentseed/asdi-examples/blob/main/examples/sentinel-1-stac-example.ipynb" rel="nofollow"><img src="https://camo.githubusercontent.com/8c5378ff3bf6f71a57442940234293bd63c7ed2418d64f74f2bda3dc6f2904ed/68747470733a2f2f73747564696f6c61622e736167656d616b65722e6177732f73747564696f6c61622e737667" alt="Open In SageMaker Studio Lab" data-canonical-src="https://studiolab.sagemaker.aws/studiolab.svg" style="max-width: 100%;"></a>
```

**Comment by jjfrench** [comment]
Just a couple of minor text suggestions in the notebook, otherwise lgtm

File: README.md (hunk 1 above)
Comment (wildintellect) [reply README.md]: I'll reword that, the same region is actually to keep requester pays costs down.

**Review by jjfrench — APPROVED** [review]
🙌🏼  Looks good!

**Pull request merged**